### PR TITLE
Release v1.0.0-rc3

### PR DESCRIPTION
**Pull request by stefanprodan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
+
+This is the third release candidate.
+
+### Improvements
+
+ - Update Helm to `v2.14.3` and kubectl to `v1.14.7`
+   [fluxcd/helm-operator#73][#73]
+ - Add EKS repo to the list of public Helm repositories in the default config
+   [fluxcd/helm-operator#64][#64]
+ - De-experimental-ise `--workers` flag
```
Comment (hiddeco) [inline CHANGELOG.md]: I think it is worth mentioning that this also increased the default amount of workers to `2`.

Comment (stefanprodan) [reply CHANGELOG.md]: The workers were set to 2 in chart since RC.1

Comment (stefanprodan) [reply CHANGELOG.md]: But ok maybe there is someone that used the YAMLs to deploy it

Comment (hiddeco) [reply CHANGELOG.md]: The chart has a dedicated changelog, this is about software changes :-)

Comment (stefanprodan) [reply CHANGELOG.md]: Done!

**Review by hiddeco — APPROVED** [review]
I moved the additional `CHANGELOG.md` change to the right commit to not bother you with the question :-).

Thanks (again) Stefan :tulip:

**Pull request merged**